lib/components/narrative/metro/departure-times-list.tsx

```diff
@@ -23,18 +20,17 @@ export const DepartureTimesList = (props: DepartureTimesProps): JSX.Element => {
   const { activeItineraryTimeIndex, itinerary, setItineraryTimeIndex } = props
   const intl = useIntl()
   const isRealTime = firstTransitLegIsRealtime(itinerary)
+  const departureLabelText = `${intl.formatTime(itinerary.startTime)} ${
+    isRealTime
+      ? intl.formatMessage({ id: 'components.StopTimeCell.realtime' })
+      : intl.formatMessage({ id: 'components.StopTimeCell.scheduled' })
```

Comment:
I believe @binh-dam-ibigroup has had notes before on doing string concatenation like this, but I forget what the note is

Reply:
It is probably ok to do telegraph style.

Reply:
I would put parentheses around the real-time/scheduled hints. That way, at least for French (and Spanish?), you could say "10:30 (real-time data)", "10:45 (theoretical schedule)", and we wouldn't have to worry about conjunctions.